lib/cloudinary.js

```diff
@@ -36,22 +36,6 @@ function chainTransformations(options, transformation=[])
   return urlOptions;
 }
 
-/**
```

Comment:
It would be better to keep the comments here, to support anyone looking at the source code.

Reply:
This is a good point regarding assisting **developers** looking at the source code.
However the Type Signatures can also serve as documentation for **end users**.
For example: go-to-definition in VSCode would re-direct to the .d.ts file if available.

My suggestion is to
- Keep the documentation as part of the TypeScript signatures.
-  **link** the source code implementation to the TypeScript signatures
   - [Example](https://github.com/cloudinary/cloudinary_npm/pull/268/files#diff-7b97ff93a95445421f186ba62c704ca5R137)
  - Note that this `@link {...}` syntax is traversal in webstorm/intellij so one can navigate to the APIs/Docs from the source code.

This pattern would enable in the future to integrate the TypeScript compiler
as a TypeChecker on the **JavaScript** code, thus adding some automated verification
that the **implementation actually matches the interface**.
- More info on this [here](https://github.com/bd82/typescript_for_public_apis)